ch_backup/backup/metadata/table_metadata.py

```diff
@@ -53,6 +55,24 @@ def get_parts(self, *, excluded_parts: Set[str] = None) -> List[PartMetadata]:
                     PartMetadata.load(self.database, self.name, part_name, raw_metadata)
                 )
 
+        if self.engine == REPLACING_MERGE_TREE:
+
+            def split_part_name(part):
+                chunks = part.split("_", maxsplit=3)
```

Comment:
Let's parse full format `{partition_id}_{min_block}_{max_block}_{level}_{mutation}`
In accordance to https://github.com/ClickHouse/ClickHouse/blob/e2821c5e8b728d1d28f9e0b98db87e0af5bc4a29/src/Storages/MergeTree/MergeTreePartInfo.cpp#L54

Reply:
And pls  add this link in the comments.

Reply:
ok